grpc/server.go

```diff
@@ -18,6 +18,8 @@ type ServerConfig struct {
 	// EnableReflection indicates whether gRPC server reflection is enabled
 	// This allows clients to introspect the server's services and methods.
 	EnableReflection bool `mapstructure:"EnableReflection"`
+	// MaxDecodingMessageSize is the maximum size of a message that the server can receive.
+	MaxDecodingMessageSize uint64 `mapstructure:"MaxDecodingMessageSize"`
```

Comment:
WDYT? No need to use uint64, since the grpc library models this as `int` (probably it would make sense to put `int` as well in the config, even though negative values don't make sense, but that way we are secured by the overflows when downcasting from `uint64/uint` to `int`). WDYT?
```suggestion
	MaxDecodingMessageSize uint `mapstructure:"MaxDecodingMessageSize"`
```

Reply:
I agree, I change it!